django/contrib/contenttypes/fields.py

```diff
@@ -253,6 +253,18 @@ def __set__(self, instance, value):
         setattr(instance, self.cache_attr, value)
 
 
+class GenericRel(ForeignObjectRel):
+    def __init__(self, field, to, related_name=None, related_query_name=None, limit_choices_to=None):
+
```

Comment:
Fixed. Some of those constructors are quite complex so I thought I'd carefully isolate each parts, but it's true it's quite foreign to this codebase and it looked funny in the few constructor that didn't really benefit from it.